ipatests/pytest_ipa/integration/tasks.py

```diff
@@ -1994,3 +1994,13 @@ def get_logsize(host, logfile):
     """ get current logsize"""
     logsize = len(host.get_file_contents(logfile))
     return logsize
+
+
+def install_package(host, pkgs):
+    dnf_cmd = '/usr/bin/dnf'
+    yum_cmd = '/usr/bin/yum'
+    result = host.run_command(['which', 'dnf'], raiseonerr=False)
+    if result.returncode == 0:
+        host.run_command([dnf_cmd, 'install', '-y', pkgs])
+    else:
+        host.run_command([yum_cmd, 'install', '-y', pkgs])
```

Comment:
I'm working on a candidate patch which will make this more generic. I'll provide it once I can get the tests to pass locally.